StripePaymentSheet/StripePaymentSheet/Source/Analytics/PaymentSheetAnalyticsHelper.swift

```diff
@@ -338,6 +385,21 @@ extension STPAnalyticsClient {
 extension PaymentSheet.Configuration {
     /// Serializes the configuration into a safe dictionary containing no PII for analytics logging
     var analyticPayload: [String: Any] {
+        var payload = commonAnalyticPayload
+        payload["payment_method_layout"] = paymentMethodLayout.description
+        return payload
+    }
+}
+
+extension EmbeddedPaymentElement.Configuration {
+    /// Serializes the configuration into a safe dictionary containing no PII for analytics logging
+    var analyticPayload: [String: Any] {
+        return commonAnalyticPayload
```

Comment:
We'll add `embeddedViewDisplaysMandateText` and `formSheetAction` in another PR?

Reply:
yep!